.circleci/config.yml

```
@@ -652,7 +652,7 @@ jobs:
             pip install --user codespell
       - run:
           name: Check spelling
-          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*" -I ./scripts/codespell_whitelist.txt
+          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*,LICENSE,codespell_whitelist.txt,codespell_ignored_lines.txt" -I ./scripts/codespell_whitelist.txt -x ./scripts/codespell_ignored_lines.txt
```

```suggestion
          command: ~/.local/bin/codespell -S "*.enc,.git,Dockerfile*,LICENSE,codespell_whitelist.txt,codespell_ignored_lines.txt" -I ./scripts/codespell_whitelist.txt --exclude-file ./scripts/codespell_ignored_lines.txt
```
We should really be using long versions of other options too.

---

https://github.com/ethereum/solidity/pull/13413